api/models/site.js

```diff
@@ -42,6 +68,16 @@ function toJSON() {
   object.createdAt = object.createdAt.toISOString();
   object.updatedAt = object.updatedAt.toISOString();
 
+  // prevent any invalid domain or demoDomain URLs from being returned
+  // or used in the creation of viewLink or demoViewLink
+  if (!isEmptyOrUrl(object.domain, false)) {
+    object.domain = null;
+  }
+
+  if (!isEmptyOrUrl(object.demoDomain, false)) {
```

Comment:
Sure, though according to @jmhooper the only values in prod or staging that aren't valid URLs are values like `"test1"`, so they can't quite be turned into valid values. We could just remove them. Do you think that would be better? I don't really like mucking around in prod databases personally, but that should be an easy enough thing to do.